src/components/SearchView.tsx

```diff
@@ -0,0 +1,163 @@
+import React, {useState} from "react"
+import { SafeAreaView, View, FlatList, Text, Image, StyleSheet, TouchableOpacity } from "react-native"
+import Business from './Business'
+
+const TEST_BUSINESSES_DATA: Business[] = [
+  {
+    id: 'bd7acbea-c1b1-46c2-aed5-3ad53abb28ba',
+    businessName: 'Starbucks Coffee',
+    distance: '1.2',
+    futureProofRating: 75,
+    userRating: 62,
+    logoURL: '../../assets/icon.png'
+  },
+  {
+    id: '3ac68afc-c605-48d3-a4f8-fbd91aa97f63',
+    businessName: 'Costa Coffee',
+    distance: '1.3',
+    futureProofRating: 88,
+    userRating: 40,
+    logoURL: '../../assets/icon.png'
+  },
+  {
+    id: '58694a0f-3da1-471f-bd96-145571e29d72',
+    businessName: 'Pret A Manger',
+    distance: '1.5',
+    futureProofRating: 85,
+    userRating: 38,
+    logoURL: '../../assets/icon.png'
+  }
+];
+
+function ratingValToColour(ratingValue: Number): string {
+  if (ratingValue > 85) {
+    return "#1EA853"
+  }
+  if (ratingValue > 70) {
+    return "#50A75C"
+  }
+  if (ratingValue > 50) {
+    return "#BED62E"
+  }
+  if (ratingValue > 30) {
+    return "#D6C52E"
+  }
+  return "brown"
+}
+
+function ratingValToText(ratingValue: Number): string {
+  if (ratingValue > 85) {
+    return "Great"
+  }
+  if (ratingValue > 70) {
+    return "Good"
+  }
+  if (ratingValue > 50) {
+    return "Okay"
+  }
+  if (ratingValue > 30) {
+    return "Bad"
+  }
+  return "Avoid"
+}
+
+const RatingCapsuleHeight = 18;
+
+const RatingCapsule = ({ratingValue}: {ratingValue: Number}) => (
+  <View style={{alignItems: "center", backgroundColor: ratingValToColour(ratingValue), width: 55, height: RatingCapsuleHeight, borderRadius: RatingCapsuleHeight/2}}>
```

Comment:
Why are you mixing using the stylesheet and defining inline styles? I think that it's only necessary for the `backgroundColour`, but the other style expressions are independent of the rating value.